src/components/SlateEditor/plugins/table/matrixHelpers.ts

```diff
@@ -91,7 +91,7 @@ export const insertCellInMatrix = (
 
 const normalizeRow = (row: TableCellElement[]) => {
   const cells: TableCellElement[] = [];
-  row.forEach((cell) => [...Array(cell?.data?.colspan)].forEach((_) => cells.push(cell)));
+  row.forEach((cell) => [...Array(cell?.data?.colspan)].forEach(() => cells.push(cell)));
```

Comment:
Trudde du måtte ha parameteret for å treffe på funksjonssignatur, men du treng altså ikkje det i typescript?

Reply:
Litt usikker på hva du mener her, tror jeg

Reply:
At forEach krever et parameter som er elementet fra lista, som heilt først i linja der du henter ut cell fra rows.